examples/dm_pg_storage.py

```diff
@@ -0,0 +1,122 @@
+# Copyright 2016 Eugene Frolov <eugene@frolov.net.ru>
+#
+# All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+import six
+
+from restalchemy.dm import filters
+from restalchemy.dm import models
+from restalchemy.dm import properties
+from restalchemy.dm import relationships
+from restalchemy.dm import types
+from restalchemy.storage.sql import engines
+from restalchemy.storage.sql import orm
+
+
+# CREATE TABLE foos (
+#      uuid CHAR(36) PRIMARY KEY,
+#      foo_field1 INT NOT NULL ,
+#      foo_field2 VARCHAR(255) NOT NULL
+# );
+class FooModel(models.ModelWithUUID, orm.SQLStorableMixin):
+    __tablename__ = "foos"
+    foo_field1 = properties.property(types.Integer(), required=True)
+    foo_field2 = properties.property(types.String(), default="foo_str")
+
+
+# CREATE TABLE bars (
+#     uuid CHAR(36) NOT NULL PRIMARY KEY,
+#     bar_field1 VARCHAR(10) NOT NULL,
+#     foo CHAR(36) NOT NULL,
+#     FOREIGN KEY (foo) REFERENCES foos(uuid) ON DELETE RESTRICT ON UPDATE
+#       RESTRICT
+# );
+#
+# CREATE INDEX rlfoo ON bars(foo);
+class BarModel(models.ModelWithUUID, orm.SQLStorableMixin):
+    __tablename__ = "bars"
+    bar_field1 = properties.property(types.String(min_length=1, max_length=10))
+    foo = relationships.relationship(FooModel)
+
+
+engines.engine_factory.configure_factory(
+    db_url="postgresql://posgres:password@127.0.0.1:5432/ra_tests"
+)
+
+# Create new foo object and store it
+foo1 = FooModel(foo_field1=10)
+foo1.save()
+
+bar1 = BarModel(bar_field1="test", foo=foo1)
+bar1.save()
+
+six.print_(list(BarModel.objects.get_all()))
```

Comment:
Let's remove `six` from examples

Reply:
The six module will be removed in the next patch from the entire library. For now let it be uniform